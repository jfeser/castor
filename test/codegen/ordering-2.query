alist(orderby([f], r1) as k, atuple([ascalar(k.f), ascalar(k.g)], cross))