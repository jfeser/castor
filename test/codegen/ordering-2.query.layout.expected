0:10 List body
0:2 Tuple body
0:1 Scalar (=(Int 1))
0:0 List len (=10)
0:0 List count (=5)
0:0 Tuple len (=2)
1:1 Scalar (=(Int 2))
2:2 Tuple body
2:1 Scalar (=(Int 1))
2:0 Tuple len (=2)
3:1 Scalar (=(Int 3))
4:2 Tuple body
4:1 Scalar (=(Int 2))
4:0 Tuple len (=2)
5:1 Scalar (=(Int 1))
6:2 Tuple body
6:1 Scalar (=(Int 2))
6:0 Tuple len (=2)
7:1 Scalar (=(Int 2))
8:2 Tuple body
8:1 Scalar (=(Int 3))
8:0 Tuple len (=2)
9:1 Scalar (=(Int 4))
